modules/cluster-entitlements.adoc

```diff
@@ -17,33 +17,22 @@
 [id="cluster-entitlements_{context}"]
 = Internet and Telemetry access for {product-title}
 
-In {product-title} {product-version}, Telemetry is the component that provides
-metrics about cluster health and the success of updates. To perform subscription
-management, including legally entitling your purchase from Red Hat, you must use
-the Telemetry service and access the
-link:https://cloud.redhat.com/openshift/install[OpenShift Infrastructure Providers] page.
+In {product-title} {product-version}, Telemetry is the component that provides metrics about cluster health and the success of updates. If your cluster is connected to the internet, Telemetry runs automatically, and your cluster is automatically entitled after installation.
 
-Because there is no disconnected subscription management, you cannot both opt
-out of sending data back to Red Hat and entitle your purchase. Support for
-disconnected subscription management might be added in future releases of
-{product-title}
+To perform subscription management, including legally entitling your purchase from Red Hat, you must access the link:https://cloud.redhat.com/openshift/install[OpenShift Infrastructure Providers] page. If you cannot send data back to Red Hat, you can perform link:https://cloud.redhat.com/openshift/register[Cluster registration] on this site.
+
+You must have internet access to:
+
+* Access the link:https://cloud.redhat.com/openshift/install[OpenShift Infrastructure Providers] page to download the installation program
+* Access link:http://quay.io[quay.io] to obtain the packages that are required to install your cluster
+* Obtain the packages that are required to perform cluster updates
+* Access link:http://cloud.redhat.com[Red Hat's software as a service page] to perform subscription management
 
 ifdef::openshift-origin,openshift-enterprise[]
 
 [IMPORTANT]
 ====
-Your machines must have direct internet access to install the cluster.
+If you do not have direct internet access, you can perform a restricted network installation on infrastructure that you provision. During that process, you download the content that is required and use it to populate a mirror registry with the packages that you need to install a cluster and generate the installation program. With some installation types, the environment that you install your cluster i will not require internet access. Before you update the cluster, you update the content of the mirror registry.
```

Comment:
Seems orthogonal to entitlements?

Reply:
The topics about internet access and entitlements.